src/app/components/search/search.js

```diff
@@ -62,7 +52,13 @@ angular
                 _.each(results, function(result){
                     _.each(result.matches, function(match){
                        if(!fileIDs.includes(result.model['unique_id'])){
-                           if((show_names && match.key === "name") || (show_descriptions && match.key === "description") || (show_columns && match.key === "columns") || (show_code && match.key === "raw_sql") || (show_tags && match.key === "tags")){
+                           if(
+                           		   (show_names && match.key === "searchableName") 
+                           		|| (show_descriptions && (match.key === "description" || match.key == "columns.description")) 
+                           		|| (show_columns && match.key === "columns.name") 
+                           		|| (show_code && match.key === "raw_sql") 
+                           		|| (show_tags && (match.key === "tags" || match.key == "columns.tags"))
+                           ) {
```

Comment:
Technically, these tickboxes no longer do exactly what they used to. They used to be "search only these fields for the query term", but are now "only return results where this field was one of the ones that included a query term". 

In real life, this means that a model tagged with `my-cool-tag` will lose out to a different model _called_ `my_cool_tag` and tagged with `something-else` when searching for `cool tag` with Tags Only ticked. 

As far as I can see, to get around this would require a re-instantiation of Fuse with a new index (by passing just the relevant keys) each time these tick boxes were toggled.